en/docs/release-notes/version-4.3.0.md

```diff
@@ -0,0 +1,34 @@
+# Ballerina VS Code extension version 4.3.0
+
+We are happy to announce the Ballerina VS Code extension 4.3.0 release with a few exciting new features and improvements. Following are the highlights of this release.
+
+- [Entity Relationship Diagram](#entity-relationship-diagram) - Helps you visualize the ER diagram for your persist model file.
+- [TOML config generator](#toml-config-generator) - Facilitates automatically generating the `Config.toml` file of your Ballerina project.
+
+If you are new to Ballerina, you can download the [installers](https://ballerina.io/downloads/) to install it. You can install the Ballerina VS Code extension from the [VS Code marketplace](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina).
+
+## New features
+
+Below are the new features introduced by this release.
```

Comment:
```suggestion

```

Reply:
We have a doc best practice to have an intro sentence after the title.